### ISSUE TITLE
utils_fit.py 里面print函数是不是有点问题

### ISSUE THREAD

**Issue by ShepherdSosimple** [bubbliiiing/deeplabv3-plus-pytorch#2]
倒数第二行代码：
print('Total Loss: %.3f || Val Loss: %.3f ' % (loss / (epoch_step + 1), val_loss / (epoch_step_val + 1)))
前面应该是total_loss吧?如下：
print('Total Loss: %.3f || Val Loss: %.3f ' % (total_loss / (epoch_step + 1), val_loss / (epoch_step_val + 1)))

**Comment by bubbliiiing** [comment]
已修改
